data/datacatalogue/molgenis.csv

```diff
@@ -32,7 +32,7 @@ Resources,,description,text,,,,,,,,,Short description of this resource
 Datasources,,type,ontology_array,,,CatalogueOntologies,DatasourceTypes,,,,,Select one or multiple datasource types
 Resources,,keywords,text,,,,,,,,,Keywords to increase findability of this resource. Try to use words that are not used in the description.
 Cohorts,Resources,,,,,,,,,,,Group of individuals sharing a defining demographic characteristic
-Cohorts,,contactEmail,,,,,,,,,,Contact e-mail address for this cohort
+Cohorts,,contactEmail,,,,,,,,,,"Contact e-mail address for this cohort. Note: this email address will be displayed in the catalogue, which is openly accessible."
```

Comment:
I don't want this change here. This is the master so this additional sentence is not needed/